src/resources/views/columns/closure_function.blade.php

```diff
@@ -0,0 +1,4 @@
+{{-- closure function --}}
+<span>
+	{!! $column['function']($entry) !!}
```

Comment:
Indent with spaces, to stay consistent with other column types (even though indentation seems all over the place in some other fields).